### PR TITLE
WIP: Add enzyme 

### PR DESCRIPTION
**Pull request by harrietc52**

### CODE REVIEW

File: src/views/Samples.vue [hunk 1]
```diff
@@ -42,22 +47,23 @@ export default {
...
+        await this.createLibrariesInTraction(selectedEnzymeId)
       } catch (error) {
         // log error
       } finally {
         this.showAlert
       }
     },
-    async createLibrariesInTraction () {
-      let sample_ids = []
+    async createLibrariesInTraction (selectedEnzymeId) {
```
Comment (stevieing) [inline src/views/Samples.vue]: Can we add a TODO to have a look at the processing in the test and see if it signifies an issue elsewhere.

File: tests/unit/views/Samples.spec.js [hunk 2]
```diff
@@ -63,39 +68,59 @@ describe('Samples.vue', () => {
...
         samples.tractionApiLibrary.data = response
         samples.tractionApiLibrary.create.mockReturnValue(response)
-        samples.createLibrariesInTraction()
+
+        let selectedEnzymeId = 1
+        samples.createLibrariesInTraction(selectedEnzymeId)
         await flushPromises()
 
-        let sample_ids = []
+        let libraryAttrs = []
```
Comment (stevieing) [inline tests/unit/views/Samples.spec.js]: Can we add a TODO to have a look at the processing in the test and see if it signifies an issue elsewhere.

File: tests/unit/views/Samples.spec.js [hunk 3]
```diff
@@ -63,39 +68,59 @@ describe('Samples.vue', () => {
...
         }
 
-        let body = { data: { type: 'libraries', attributes: { libraries: sample_ids }}}
+        let body = { data: { type: 'libraries', attributes: { libraries: libraryAttrs }}}
         expect(samples.tractionApiLibrary.create).toBeCalledWith(body)
         expect(samples.message).toEqual("Something went wrong")
       })
     })
 
   })
```
Comment (stevieing) [inline tests/unit/views/Samples.spec.js]: Are these commented out methods still needed?

Comment (harrietc52) [reply tests/unit/views/Samples.spec.js]: Nope :)

**Pull request merged**